locale-fork/locale/de/LC_MESSAGES/django.po

```diff
@@ -3828,7 +3845,7 @@ msgstr ""
 #. Translators: KOSMO
 #: apps/users/emails.py:18
 msgid "It does not correspond to the {}netiquette{}."
-msgstr ""
+msgstr "Er entspricht nicht der Netiquette."
```

Comment:
This needs the curly brackets {}, otherwise there will be no link in the translated string.